### PR TITLE
Add Docker Compose Project for DICOM Server in Visual Studio

### PR DESCRIPTION
**Pull request by wsugarman**
## Description
- Add DICOM Server docker compose project
- Refactor Dockerfiles and folder structure
- Remove dependency on submodule now that Azurite Docker container supports table storage

## Related issues
[AB#82320](https://microsofthealth.visualstudio.com/Health/_workitems/edit/82320)

## Testing
Ensure build pipelines pass

### CODE REVIEW

File: src/Microsoft.Health.Dicom.Web/Microsoft.Health.Dicom.Web.csproj [hunk 1]
```diff
@@ -39,4 +40,4 @@
     </VisualStudio>
   </ProjectExtensions>
 
-</Project>
+</Project>
```
Comment (wsugarman) [inline src/Microsoft.Health.Dicom.Web/Microsoft.Health.Dicom.Web.csproj:43]: Oops, I removed the newline

File: samples/docker/docker-compose.yaml [hunk 2]
```diff
@@ -12,61 +12,52 @@ services:
...
-    build: 
+    build:
       context: ./../../
-      dockerfile: ./build/docker/sql/Dockerfile
+      dockerfile: ./docker/sql/Dockerfile
     restart: always
     environment:
       SA_PASSWORD: ${SAPASSWORD}
       ACCEPT_EULA: "Y"
-    # # uncomment if you want to expose sql management studio against localhost
```
Comment (KnicKnic) [inline samples/docker/docker-compose.yaml:46]: ># uncomment [](http://example.com/codeflow?start=6&length=11)

Normally I leave double comment to signal that it should stay commented if you "uncomment" it

File: docker/docker-compose.vs.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      SqlServer__ConnectionString: "Server=tcp:sql,1433;Initial Catalog=DICOM;Persist Security Info=False;User ID=sa;Password=T3stP@ssw0rd;MultipleActiveResultSets=False;Connection Timeout=30;"
+    user: root
+    ports:
+      - "63839:8080"
+    volumes:
+      - ${APPDATA}/ASP.NET/Https:/root/.aspnet/https:ro
+  sql:
+    environment:
+      # [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Password is only used for development.")]
+      SA_PASSWORD: T3stP@ssw0rd
```
Comment (KnicKnic) [inline docker/docker-compose.vs.yml:20]: >T3stP@ssw0rd [](http://example.com/codeflow?start=19&length=12)

may want default values for these
[https://github.com/compose-spec/compose-spec/blob/master/spec.md#interpolation](https://github.com/compose-spec/compose-spec/blob/master/spec.md#interpolation)

**Review by KnicKnic — APPROVED** [review]
:shipit:

File: docker/docker-compose.vs.yml (hunk 3 above)
Comment (wsugarman) [reply docker/docker-compose.vs.yml]: Makes sense!

File: samples/docker/docker-compose.yaml (hunk 2 above)
Comment (wsugarman) [reply samples/docker/docker-compose.yaml:46]: Oh ok. I'll add them back

**Pull request merged**